modules/apps/27-interchain-accounts/types/expected_keepers.go

```diff
@@ -36,3 +36,10 @@ type PortKeeper interface {
 	BindPort(ctx sdk.Context, portID string) *capabilitytypes.Capability
 	IsBound(ctx sdk.Context, portID string) bool
 }
+
+type ScopedKeeper interface {
```

Comment:
Could we add a godoc? Just something short and sweet like the others would be perfect <3

Reply:
```suggestion
// ScopedKeeper defines the expected x/capability scoped keeper interface
type ScopedKeeper interface {
```